src/components/signupForm/SignupForm.js

```diff
@@ -47,11 +47,37 @@ class SignupForm extends Component {
 
     render() {
         const { classes } = this.props;
+
+        /*ECS:
+
+        The onChange attached to the <form> tag should be on each individual <TextField> leaving only
+        the onSubmit on the <form> tag itself
+
```

Comment:
What's the reasoning behind that? Isn't it best to achieve the goal in less code?